.github/workflows/db_rollback.yml

```diff
@@ -0,0 +1,68 @@
+name: Rollback DB
+
+on:
+  workflow_dispatch:
+    inputs:
+      rollbackCount:
+        description: 'Number of migrations to rollback'
+        required: true
+        default: # of rollback you require in the file input
+      rollbackFile:
+        description: "File with migrations you want to rollback"
+        required: true
+        default: Filepath to yml
+
+jobs:
+  terraform-deploy:
+    name: Terraform Deploy
+    environment:
+      name: ${{ inputs.ENVIRONMENT }}
+    runs-on: ubuntu-latest
+    env:
+      ARM_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
+      ARM_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+      ARM_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
```

Comment:
We have different secrets depending on whether we are deploying to the Flexion or CDC domain.  Maybe we can do a ternary if statement based on the workflow dispatch environment input?

Something like `${{ github.event.inputs.environment == 'internal' ? secrets.AZURE_CLIENT_ID : secrets.AZURE_CDC_CLIENT_ID }}`.